src/Assets/css/profile.css

```diff
@@ -0,0 +1,41 @@
+.profile-wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  padding: 10px;
+}
+
+.missions,
+.rockets {
+  width: calc(50% - 5px);
```

Comment:
- [ ] This width is making your page shrink in all views, you can make it width 100% or simply remove the width style
![Screenshot from 2023-05-16 20-12-01](https://github.com/Rimuru-Tempest-26/React_Group_Project/assets/66097371/4311f756-1d5d-4355-aa3b-129ab42a1707)

Reply:
Fixed.